docs/connectors/webexteams.md

```diff
@@ -0,0 +1,23 @@
+# opsdroid connector webex teams
+
+A connector for [opsdroid](https://github.com/opsdroid/opsdroid) to interact with [Webex Teams](https://www.webex.com/team-collaboration.html).
+##### Note: Webex Teams was used to be called cisco spark. We changed cisco spark connector to webex teams connector 
```

Comment:
Let's make this italic rather than a title
```suggestion
_Note: Webex Teams was used to be called cisco spark. We changed cisco spark connector to webex teams connector _
```